pkg/checks/traceroute/config.go

```diff
@@ -37,13 +37,12 @@ func (c *Config) Validate() error {
 
 	for i, t := range c.Targets {
 		ip := net.ParseIP(t.Addr)
-
 		if ip != nil {
 			continue
 		}
 
 		_, err := url.Parse(t.Addr)
-		if err != nil && ip == nil {
+		if err != nil {
 			return checks.ErrInvalidConfig{CheckName: CheckName, Field: fmt.Sprintf("traceroute.targets[%d].addr", i), Reason: "invalid url or ip"}
```

Comment:
shouldn't this be err != nil || ip == nil, in case the IP is really nil after the `net.ParseIP`?

Reply:
I've removed this because my linter complained and he's right. We're checking before if it's not nil:
https://github.com/caas-team/sparrow/blob/bed33794e600adc70b3e6a89f29557a7ddbdfe0a/pkg/checks/traceroute/config.go#L39-L42

So at this point `ip` is always nil.